src/js/providers/html5.js

```diff
@@ -482,7 +482,7 @@ define([
                 // Playback rate is broken on Android HLS
                 _this.supportsPlaybackRate = false;
             }
-            if (source.preload && source.preload !== _videotag.getAttribute('preload')) {
+            if (source.preload !== _videotag.getAttribute('preload')) {
                 _setAttribute('preload', source.preload);
```

Comment:
Are you sure `source.preload` is set correctly? I see you setting it on the item level, not the source level.